docs/user_guide.md

```diff
@@ -719,6 +721,200 @@ If you are a project admin, you can disable a Robot Account by clicking "Disable
+
+In this case, all of the images in repository A are retained because they have been pulled in the last 7 days. None of the images in repositories B to E are retained, because none of them has been pulled in the last week. In this example, 100 images are retained, as opposed to 34 images in example 1.
+
+### Combining Rules on a Respository
+
+You can define up to 15 rules per project. You can apply multiple rules to a repository or set of repositories. When you apply multiple rules to a repository, they are applied with `OR` logic rather than with `AND` logic. In this way, there is no prioritization of application of the rules on a given repository. 
+
+#### Example 3
+
+This example uses the same project and repositories as examples 1 and 2, but sets two rules:
+
+- Rule 1: Retain all of the images in each repository that have been pulled in the last 7 days.
+- Rule 2: Retain a maximum number of 10 images in each repository.
+
+In this case, repository A only retains the 10 most recent images, even though images 11 to 100 have been pulled in the last week. Repositories B to E retain no images, because none of those images have been pulled in the last week. In this example, 10 images are retained, as opposed to 34 images in example 1 and 100 in example 2.
+
+
+#### Example 4
+
+This example uses a different repository to the previous examples.
+
+- You have a repository that has 12 tags:
+
+  |Production|Release Candidate|Release|
+  |---|---|---|
+  |`2.1-your_repo-prod`|`2.1-your_repo-rc`|`2.1-your_repo-release`|
+  |`2.2-your_repo-prod`|`2.2-your_repo-rc`|`2.2-your_repo-release`|
+  |`3.1-your_repo-prod`|`3.1-your_repo-rc`|`3.1-your_repo-release`|
+  |`4.4-your_repo-prod`|`4.4-your_repo-rc`|`4.4-your_repo-release`| 
+
+- You define three tag retention rules on this repository:
+  - Retain the 10 most recently pushed image tags that start with `2`.
+  - Retain the 10 most recently pushed image tags that end with `-prod`.
+  - Exclude the image tag `2.1-your_repo-prod` from the list of tags to retain.
+
+In this example, the rules are applied to the following 7 tags:
+
+- `2.1-your_repo-rc`
+- `2.1-your_repo-release`
+- `2.2-your_repo-prod`
+- `2.2-your_repo-rc`
+- `2.2-your_repo-release`
+- `3.1-your_repo-prod`
+- `4.4-your_repo-prod`
+
+### How Tag Retention Rules Interact with Project Quotas
+
+The system administrator can set a maximum on the number of artifacts that a project can contain and the amount of storage that it can consume. For information about project quotas, see [Configure Project Quotas](LINK). 
+
+Project quota settings take precedence over tag retention rules. If you set a tag retention rule to keep the 5 most recent images in a repository, but the 5th image takes the total amount of storage consumed by the project above the project quota for storage, the 5th image will be deleted.  

```

Comment:
I don't think this will happen because retention policy only trims down the repo, in other words, running tag retention will not violate any quotas set that was not already in violation of to begin with.  Does that make sense? @stuclem

Reply:
Reworded as follows in https://github.com/goharbor/harbor/pull/8869/commits/38092ca3d28b250ea29e305bb264417c0125a63c:

"Project quota settings take precedence over tag retention rules. If you set a quota on a project, this quota cannot be exceeded. The quota is applied to a project even if you set a retention rule that would exceed it. In other words, you cannot use retention rules to bypass quotas."

Reply:
I think this is fine for now, I'm just a little hesitant to define the precedence because it can get tricky.  I think 'cannot use retention rules to bypass quotas' captures it very well.  LGTM for now, thanks for the thoughtful wording.

Reply:
@xaleeks is this OK now? ^^^^

Reply:
@xaleeks I deleted "Project quota settings take precedence over tag retention rules" and left the rest as it was.

Reply:
lgtm